packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.types.ts

```diff
@@ -143,18 +145,12 @@ export interface IBaseFloatingPickerProps<T> extends React.ClassAttributes<any>
   suggestionItems?: T[];
 }
 
-export interface IBaseFloatingPickerSuggestionProps {
-  /**
-   * Whether or not the first selectable item in the suggestions list should be selected
-   */
-  shouldSelectFirstItem?: () => boolean;
-
-  /**
-   * The header items props
-   */
-  headerItemsProps?: ISuggestionsHeaderFooterProps[];
-  /**
-   * The footer items props
-   */
-  footerItemsProps?: ISuggestionsHeaderFooterProps[];
-}
+/**
+ * Props which are passed on to the inner Suggestions component
+ *
+ * TODO (adjective-object) remove the ' = any' before the next major version cut.
+ */
+export type IBaseFloatingPickerSuggestionProps<T = any> = Pick<
+  ISuggestionsControlProps<T>,
+  'shouldSelectFirstItem' | 'headerItemsProps' | 'footerItemsProps' | 'showRemoveButtons'
+>;
```

Comment:
The type `T` doesn't actually seem necessary for any of the picked properties, so I don't think we lose any type safety by not exposing it. As a result, I'm wondering if this could instead be:
```tsx
export type IBaseFloatingPickerSuggestionProps = Pick<
  ISuggestionsControlProps<any>,
  'shouldSelectFirstItem' | 'headerItemsProps' | 'footerItemsProps' | 'showRemoveButtons'
>;
```

Reply:
What do you think @Adjective-Object?

Reply:
Oh, good catch!